modules/WordCloudPlugin/README.md

```diff
@@ -0,0 +1,6 @@
+## Word Cloud Plugin
```

Comment:
Technically speaking, you are not showing a World Cloud at the moment. I guess that's still the intention for a future update? If not, maybe consider renaming the plugin

Reply:
Ahhh... Well... This is both an intent (a future version might display words on two axes), and a way to make the function of the plugin immediately clear: summarizing an unstructured text by showing its most frequent words.